inc/classes/admin/settings/class-page.php

```diff
@@ -186,7 +186,14 @@ public function render_page() {
 
 		$this->render->set_hidden_settings( $this->settings->get_hidden_settings() );
 
-		echo $this->render->generate( 'page', [ 'slug' => $this->slug, 'btn_submit_text' => $rocket_valid_key ? __( 'Save Changes', 'rocket' ) : __( 'Validate License', 'rocket' ) ] );
+		$btn_submit_text = $rocket_valid_key ? __( 'Save Changes', 'rocket' ) : __( 'Validate License', 'rocket' );
+		echo $this->render->generate( // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Dynamic content is properly escaped in the view.
+			'page',
+			[
+				'slug'            => esc_attr( $this->slug ),
+				'btn_submit_text' => esc_html( $btn_submit_text ),
```

Comment:
PHPCS complains in here too, so there are 2 solutions :
1. Either add another phpcs:ignore for OutputNotEscaped 
2. Escape these 2 vars.
In case case, PHPCS will complain also in /views files if these are not escaped.

So these 2 are escaped twice